nemo/collections/asr/parts/submodules/conformer_modules.py

```diff
@@ -190,6 +210,19 @@ def forward(self, x, pad_mask=None):
         x = x.transpose(1, 2)
         return x
 
+    def reset_parameters_conv(self):
```

Comment:
Isn't it better to call the method just reset_parameters to be consistent?
The same for the FF module.